middleware/strip_test.go

```diff
@@ -80,6 +83,12 @@ func TestStripSlashesInRoute(t *testing.T) {
 	if _, resp := testRequest(t, ts, "GET", "/hi/", nil); resp != "nothing here" {
 		t.Fatalf(resp)
 	}
+	if _, resp := testRequest(t, ts, "GET", "/accounts/admin", nil); resp != "accounts index" {
+		t.Fatalf(resp)
+	}
+	if _, resp := testRequest(t, ts, "GET", "/accounts/admin/", nil); resp != "accounts index" {
```

Comment:
I think `GET /accounts/admin` should work for sure, being handled by `GET /` in the subrouter.

Do we need to support `/accounts/admin/` too? I think it doesn't really hurt, and makes sense ... since `""` and `"/"` from the sub-router perspective should be the same.